src/draw.c

```diff
@@ -543,12 +543,13 @@ view_column_draw(struct view *view, struct line *line, unsigned int lineno)
 		{
 			enum line_type type = line->type;
 			const char *text = column_data.text;
+			size_t indent;
```

Comment:
Shouldn't this be initialized since we only set it to `get_graph_indent(text)` if `line->graph_indent` is non-zero?
```suggestion
			size_t indent = 0;
```

Reply:
Both the initialisation and the use are conditioned by `line->graph_indent` but, agreed, it never hurts to initialise a variable in case someone wants to use it in the future and misses this condition.